Packs/Cymulate/Integrations/Cymulate_v2/Cymulate_description.md

```diff
@@ -0,0 +1,3 @@
+configuring the integration
```

Comment:
remove this file, it should solve the build issue (`[Errno 2] No such file or directory: 'Packs/Cymulate/Integrations/Cymulate_v2/Cymulate.yml'`)